pkgs/development/libraries/jellyfin-ffmpeg/default.nix

```diff
@@ -0,0 +1,29 @@
+{ ffmpeg, ffmpeg-full, fetchFromGitHub, lib }:
+(ffmpeg-full.override { ffmpeg = ffmpeg; }).overrideAttrs (old: rec {
+  name = "jellyfin-ffmpeg";
+  version = "4.4.1-1";
+
+  src = fetchFromGitHub {
+    owner = "jellyfin";
+    repo = "jellyfin-ffmpeg";
+    rev = "883570db52d230f7f9ab6c532069002d7a5ed90c";
+    sha256 = "0fy0d1z5nk077ayavvx7anb2b45kva0bkzajfbjnjv3arv6bhh1l";
+  };
+
+  prePatch = ''
+    for file in $(cat debian/patches/series); do
+      patch -p1 < debian/patches/$file
+    done
+
+    ${old.prePatch}
+  '';
+
+  doCheck = false; # https://github.com/jellyfin/jellyfin-ffmpeg/issues/79
```

Comment:
I am not overly familiar with the `ffmpeg` codebase, there are several tests failing due to these outdated reference values. Feel free to suggest any clean ways to disable just the failing tests.